src/Sylius/Behat/Context/Api/Admin/ManagingPromotionsContext.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ResponseCheckerInterface;
+use Sylius\Component\Core\Model\PromotionInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingPromotionsContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $apiAdminClient;
```

Comment:
As we are in the admin context:
```suggestion
    private $client;
```
?